payloads/library/general/Uptime/payload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Title:		Uptime
+# Description:		The uptime payload displays the current time, the length of time the system has been up, the number of users, and the load average of the system over the last 1, 5, and 15 minutes.
+# Author:		Moliata
+# Version:		1.0
+# Category:		general
+# Target:		Packet Squirrel by Hak5
+# Net Mode:		TRANSPARENT
+function start()
+{
+        LED SETUP
+        NETMODE TRANSPARENT
+        mkdir -p /mnt/loot/uptime
+        sleep 5
+}
+function run() {
+        LED ATTACK
+        while true
+                do
+                        BUTTON && {
+                                uptime >> /mnt/loot/uptime/uptime.log
```

Comment:
You should make this a single '>' command so the file is overwritten every time, if not this file could grow infinitely large.

Reply:
That's a great catch, will change it.

Reply:
ah sorry!

I thought it automatically wrote over and over again.

Still, what is the purpose of appending rather than overwriting?

Reply:
I still think this payload should use the single '>' command, uptime refers to the total time a machine has been running and I dont believe its worth knowing previous uptime executions.  Unless you can think of any reason?

Reply:
Awesome!